graphics/fontforge/Portfile

```diff
@@ -1,17 +1,19 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem      1.0
-
-name            fontforge
-version         20120731
-revision        3
-set docversion  ${version}
-set ftversion   2.4.10
-categories      graphics fonts
-platforms       darwin
-maintainers     nomaintainer
-license         BSD
-description     Outline and bitmap font editor/converter for many formats
+PortSystem          1.0
```

Comment:
Total nitpicking, please ignore :)
I suggest to make whitespace change first. (In principle I can do that in master and later fix this branch.)

Reply:
Weird, my editor should have used spaces in the tcl file. Maybe I pasted a tab or something. I'll fix it.